sources/packages/web/src/views/institution/locations/confirmation-of-enrollment/LocationCOESummary.vue

```diff
@@ -41,9 +41,24 @@
         <c-o-e-summary-data
           :locationId="locationId"
           :enrollmentPeriod="EnrollmentPeriod.Upcoming"
-          header="Upcoming enrolment"
-          subTitle="These applications are still outside of the 21 days of the study start date."
-        />
+          header="Upcoming and
+        Previous Enrolments"
+        >
+          <template #coeSummarySubtitle>
```

Comment:
I would prefer to keep the slot name and propname as same. 

Currently the prop name is `subtitle` and slot name is `coeSummarySubtitle`